SCons/Script/SConsOptions.py

```diff
@@ -39,7 +39,7 @@
 
 diskcheck_all = SCons.Node.FS.diskcheck_types()
 
-experimental_features = {'warp_speed', 'transporter'}
+experimental_features = {'warp_speed', 'transporter', 'ninja'}
```

Comment:
are you intending to leave the two "dummy" names in now there's a real one?

Reply:
Yes. Easter eggs are fun?